packages/main/src/MultiComboBoxItem.hbs

```diff
@@ -1,7 +1,7 @@
 {{>include "./ComboBoxItem.hbs"}}
 
 {{#*inline "listItemContent"}}
-	<ui5-checkbox ?checked="{{selected}}" tabindex="invalid" accessible-name="{{_accessibleName}}"></ui5-checkbox>
+	<ui5-checkbox ?disabled="{{_readonly}}" ?checked="{{selected}}" tabindex="invalid" accessible-name="{{_accessibleName}}"></ui5-checkbox>
```

Comment:
what is the purpose of this? readonly MCB has no possibility to open the picker

Reply:
<img width="648" alt="image" src="https://github.com/user-attachments/assets/1c97a28a-34fa-46b4-9cd3-a4367145e3a6">

n more